### PR TITLE
WIP: Devtools prompt

### PR DESCRIPTION
**Pull request by paulrouget**
Depends on #25242

Ask the user before accepting devtools incoming connection.

### CODE REVIEW

File: support/hololens/ServoApp/ServoControl/Servo.cpp [hunk 1]
```diff
@@ -119,7 +119,7 @@ Servo::Servo(hstring url, hstring args, GLsizei width, GLsizei height,
 
   // Example Call when *pfilters[] is used:
   // o.vslogger_mod_list = pfilters; // servo log modules
-  // o.vslogger_mod_size = sizeof(pfilters) / sizeof(pfilters[0]) -1; //
+  // o.vslogger_mod_size = sizeof(pfilters) / sizeof(pfilters[0]); //
```
Comment (asajeffrey) [inline support/hololens/ServoApp/ServoControl/Servo.cpp]: Not sure why this has changed?

File: support/hololens/ServoApp/Package.appxmanifest [hunk 2]
```diff
@@ -27,26 +27,24 @@
...
     </Application>
   </Applications>
   <Capabilities>
     <Capability Name="internetClient" />
     <Capability Name="codeGeneration" />
     <Capability Name="privateNetworkClientServer" />
+    <Capability Name="internetClientServer" />
   </Capabilities>
-</Package>
+</Package>
```
Comment (asajeffrey) [inline support/hololens/ServoApp/Package.appxmanifest]: Are these changes just changes to the XML formatting?

**Review by asajeffrey — APPROVED** [review]
The important changes look good, should this wait till #25242 lands?

File: support/hololens/ServoApp/ServoControl/Servo.cpp (hunk 1 above)
Comment (paulrouget) [reply support/hololens/ServoApp/ServoControl/Servo.cpp]: I realized that it was wrong. The last filter was never used.

File: support/hololens/ServoApp/Package.appxmanifest (hunk 2 above)
Comment (paulrouget) [reply support/hololens/ServoApp/Package.appxmanifest]: Visual Studio doing its things.

**Comment by paulrouget** [comment]
Landed as part of #25714 25242

**Pull request closed without merging**